backend/app/adapter/google/account_integration_test.go

```diff
@@ -0,0 +1,91 @@
+package google
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/short-d/app/mdtest"
+	"github.com/short-d/short/app/entity"
+)
+
+func TestAccount_GetSingleSignOnUser(t *testing.T) {
+	testCases := []struct {
+		name            string
+		httpResponse    *http.Response
+		httpErr         error
+		expectHasErr    bool
```

Comment:
`expectHasErr` equals to `true` only when `httpErr` is not `nil`. Maybe we can replace these 2 variables with one variable `expectedErr`?

Also, you can use [errors.Is](https://golang.org/pkg/errors/#Is)

Reply:
Currently, we don't have a error types defined for account API errors. I wonder whether we can still use`Is` here. I am reading [this](https://blog.golang.org/go1.13-errors) blog.

Reply:
can we add something like assert.hasErr(), assert.NoErr()

Reply:
@dawntomm Unfortunately no. This is prohibited by Google's Go coding best practices.